### PR TITLE
Fixes his grace awakened overlays

### LINKED ISSUE #25570: when His Grace is shaking, the blood on him does not shake as well
plz fix

### PR DESCRIPTION
**Pull request by Cyberboss**
Fixes #25570

### CODE REVIEW

File: code/game/objects/items/weapons/his_grace.dm [hunk 1]
```diff
@@ -125,7 +125,29 @@
...
+        M2.translate(7,23)
+        var/matrix/M3 = matrix(src)
+        M3.translate(8,22)
+        var/matrix/M4 = matrix(src)
+        M4.translate(7,21)
+        transforms = list(M,M2,M3,M4)
+
+    animate(src, transform = transforms[1], time = timings[1], loop = -1)
+    for(var/i in 2 to 4)
+        animate(src, transform = transforms[i], time = timings[i])
```
Comment (RemieRichards) [inline code/game/objects/items/weapons/his_grace.dm]: NO, I said no src for the loop ones, do you even ```animate()```?

File: code/game/objects/items/weapons/his_grace.dm [hunk 2]
```diff
@@ -125,7 +125,29 @@
...
 	playsound(user, 'sound/effects/pope_entry.ogg', 100)
-	icon_state = "his_grace_awakened"
+	DoAnimation()
+
+/obj/item/weapon/his_grace/proc/DoAnimation(start)
+    var/static/list/timings = list(
+    2,
+    1,    
+    2,
+    3)
```
Comment (ChangelingRain) [inline code/game/objects/items/weapons/his_grace.dm]: this is ten times slower, as his grace's icon state had animation times of 0.2/0.1/0.2/0.3 deciseconds.

Comment (RemieRichards) [inline code/game/objects/items/weapons/his_grace.dm]: don't blame me, cyberboss gave me bad numbers.

Comment (coiax) [inline code/game/objects/items/weapons/his_grace.dm]: shouldn't this list only be created if null?

Comment (RemieRichards) [inline code/game/objects/items/weapons/his_grace.dm]: re-read @coiax

**Comment by Cyberboss** [comment]
```
[16:33:21] Runtime in his_grace.dm,138: Invalid matrix
  proc name: DoAnimation (/obj/item/weapon/his_grace/proc/DoAnimation)
  usr: Nova Woolery (guest2259647179) (/mob/living/carbon/human)
  usr.loc: The floor (110,141,1) (/turf/open/floor/plasteel)
  src: His Grace (/obj/item/weapon/his_grace)
  src.loc: Nova Woolery (/mob/living/carbon/human)
  call stack:
  His Grace (/obj/item/weapon/his_grace): DoAnimation(null)
  His Grace (/obj/item/weapon/his_grace): awaken(Nova Woolery 
```

**Comment by RemieRichards** [comment]
could of swore ```matrix(ATOM)``` was a shortcut for ```matrix(ATOM.transform)```, just add ```.transform``` onto the ```src``` (and by that I mean drop the ```src```s altogether and just use ```transform```)

File: code/game/objects/items/weapons/his_grace.dm (hunk 2 above)
Comment (coiax) [inline code/game/objects/items/weapons/his_grace.dm]: I still don't follow, won't the list be recreated every time the proc is called?

Comment (RemieRichards) [inline code/game/objects/items/weapons/his_grace.dm]: ```/static```

**Comment by Cyberboss** [comment]
Throwing the damn thing stops the animation TT

**Comment by RemieRichards** [comment]
@Cyberboss make it no_spin? it's the spin animation's fault.

**Comment by Cyberboss** [comment]
The laziness overtake me

**Pull request closed without merging**